test/__fixtures__/workspace/list-error2.txt

```diff
@@ -0,0 +1,5 @@
+You have not selected any the of following - Project
```

Comment:
doesn't make sense. error doesn't show Org, but Org and config shows is 'no org selected'